New Pull Request for Adding Individual ID Number

trustedcomputer opened the pull request:
I've discarded my commits and synced with upstream, then added the commits again.

trustedcomputer committed:
Update achgeneration.md

Add Individual ID Number documentation

trustedcomputer committed:
Update check_run_settings.json

Add field for Individual ID Number From to select the source as None, Naming Series, or Party Name

trustedcomputer committed:
Update check_run.py

Add logic to use choice made in check_run settings to assign the appropriate value to individual_id_number

agritheory commented:
Hey Ben, I'll be able to take a look at this PR this week, sorry for the delay.

trustedcomputer committed:
Fix syntax errors

The new if/elif/else clauses for individual_id_number_from around lines 687-694 were incorrectly pasted into the prior exceptions.append parenthesis

trustedcomputer commented:
Thanks, @agritheory. Once this one is done, I have another one, quick and easy, to fix another issue. But I discovered if I commit changes to my personal fork, they become part of the existing PR. So I'll wait on that.

agritheory reviewed (approved):
@trustedcomputer Ben this came out great!
